src/Scripts/Resource.gd

```diff
@@ -48,6 +56,8 @@ func enabled_click_child():
 	for key in dict_adjancent_building_positions.keys():
 		dict_adjancent_building_positions[key].enable_click()
 
+	sound_effect($MainBuildingReady, 0.15)
```

Comment:
tipically you do this at the top like `onready var main_building = $MainBuildingReady` otherwise your are requiring that node every time the fucntion is invoked